frontend/components/GameSpoilersList.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+// eslint-disable-next-line no-unused-vars
+import {AppStore} from '../storage/common';
+// eslint-disable-next-line no-unused-vars
+import { GameSpoilers } from '../storage/models/GameSpoilers';
+interface GameSpoilersListProps{
+	gameSpoilers: Map<number, GameSpoilers>;
+}
+const GameSpoilersList: React.FC<GameSpoilersListProps> = (props: GameSpoilersListProps) =>{
```

Comment:
It could probably be called differently - `GameSpoilerConfig` or `GameSpoilerEditor` or even just `GameSpoilerEditableList` (though probably you want `GameAvailability<whatever>` since I suggested changing spoiler to something else earlier)